scripts/lib/Utils.psm1

```diff
@@ -244,6 +244,10 @@ Function registerTest($testname, $index, $bucket, $filter, $moreParams, $cluster
           $testWeight = $weight
         }
 
+        If ($vst) {
+          $testparams = $testparams + " --vst true"
```

Comment:
Is this option silently ignored in ArangoDB versions which do not know it?

Reply:
yes, testing.js will add it to `options` which in term will not be evaluated.